caffe2/proto/CMakeLists.txt

```diff
@@ -4,5 +4,6 @@ file(GLOB Caffe2_PROTOBUF_FILES "${CMAKE_CURRENT_SOURCE_DIR}/*.proto")
 
 PROTOBUF_GENERATE_CPP(Caffe2_PROTO_SRCS Caffe2_PROTO_HEADERS ${Caffe2_PROTOBUF_FILES})
 
-add_library(Caffe2_PROTO ${Caffe2_PROTO_HEADERS} ${Caffe2_PROTO_SRCS})
+add_library(Caffe2_PROTO SHARED ${Caffe2_PROTO_HEADERS} ${Caffe2_PROTO_SRCS})
```

Comment:
I was wondering about this - I'd normally build everything as shared and link it, but if you'd prefer static I can do that (and I realise that's what you do with BREW)